### ISSUE TITLE
Stuck detecting build env with v0.8

### ISSUE THREAD

**Issue by s-martin** [vector-of-bool/vscode-cmake-tools#79]
With the new version 0.8.0 the detection of build env is stuck (see picture). 
This blocks all further CMake steps

![grafik](https://cloud.githubusercontent.com/assets/6143783/20381781/4a234352-aca9-11e6-8d7a-d5dae572688f.png)

I'm using
CentOS 7.2
GCC 4.8
CMake 2.8.11

**Comment by filipsky** [comment]
The same situation here, I'm using:
Ubuntu 16.10
GCC 6.2.0
CMake 3.5.2

is there there some log file I can provide to you?

**Comment by magpier84** [comment]
The same:
OSX 10.11.6
Clang 3.9
CMake 3.5.2

**Comment by s-martin** [comment]
@magpier84 What did you do to make it working?

**Comment by magpier84** [comment]
I restarted VSCode and removed all files from the build directory.

**Comment by s-martin** [comment]
@magpier84 OK, the restart did the trick.

Deleting build directory and restarting also worked for me

**Comment by vector-of-bool** [comment]
Argh!! I didn't test on my Linux box and it's now happening to me as well!

**Comment by filipsky** [comment]
It's not limited to only Linux, happens on macOS as well.
But as @magpier84 mentioned, the build dir delete and vscode restart unfreazes the extension

**Comment by vector-of-bool** [comment]
Bah... This is unacceptable and doesn't even seem to be working on my other machines. Somehow it worked well without a hitch on my dev box... I'll have to roll this back ASAP. Unfortunately, I'm away from home and can't get to it for a few hours...

**Comment by bbosnjak** [comment]
I have same problem with every restart. I have to I removed all files from the build with every restart!
I am using windows and crosscompiling:
![image](https://cloud.githubusercontent.com/assets/3226555/20403141/e81e1860-acff-11e6-8ea0-fce635865759.png)

**Comment by Lillecarl** [comment]
I get the same error:
https://imgur.com/GlWIpn6

```
$ clang --version
Apple LLVM version 8.0.0 (clang-800.0.42.1)
Target: x86_64-apple-darwin16.1.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

 $ cmake --version
cmake version 3.7.0

$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.12.1
BuildVersion:	16B2555
```

**Comment by vector-of-bool** [comment]
There! Should now be fixed!

**Comment by s-martin** [comment]
Fixed in 0.8.1. 

Thanks for the extremely fast response time!
